lib/pbench/test/unit/server/test_indexing_tarballs.py

```diff
@@ -272,6 +273,35 @@ def find_dataset(self, resource_id: str):
         )
 
 
+class FakeAudit:
+    BACKGROUND_USER = "test"
+    audits: list["FakeAudit"] = []
+    sequence: int = 1
+
+    def __init__(self, **kwargs):
+        for k, v in kwargs.items():
+            setattr(self, k, v)
+        if not hasattr(self, "id"):
+            self.id = self.sequence
+            __class__.sequence += 1
+
+    def as_json(self) -> JSONOBJECT:
+        return {k: v for k, v in self.__dict__.items()}
```

Comment:
I'm not sure it actually is a normal `dict`, and I'm not sure what the straight `dict` constructor would do, because I *do* want to return a normal "JSON" `dict`. Yeah, just passing the list of `.items()` tuples would probably work, but that's not worth changing at this point.

Reply:
> I'm not sure what the straight dict constructor would do

It's pretty accommodating, and it's happy to accept the output of `.items()` as input.